etc/kayobe/pulp-repo-versions.yml

```diff
@@ -33,4 +39,12 @@ stackhpc_pulp_repo_rocky_8_7_baseos_version: 20221202T032715
 stackhpc_pulp_repo_rocky_8_7_extras_version: 20221201T192704
 stackhpc_pulp_repo_rocky_8_7_nfv_version: 20221202T032715
 stackhpc_pulp_repo_rocky_8_7_powertools_version: 20221202T032715
-stackhpc_pulp_repo_mlnx_ofed_5_7_1_0_2_0_rhel8_6_version: 20220920T151419
+stackhpc_pulp_repo_rocky_9_appstream_version: 20230228T044432
+stackhpc_pulp_repo_rocky_9_baseos_version: 20230228T044432
```

Comment:
These need to include the minor version, similar to `stackhpc_pulp_repo_rocky_8_7_powertools_version`

Reply:
Done